### ISSUE TITLE
inherited roles with mutations

### ISSUE THREAD

**Issue by manuFL** [hasura/graphql-engine#7071]
Hi, I've made a role with only update permission on a table, and another with only elect permission over the same table.

If I set one of them at time they work, but if I combine them into another role with role inheritance i cannot get the mutation work.

Are inherited roles working only for selects? If yes, do you confirm that they break all the mutation permissions?

Thank you in advance

**Comment by coco98** [comment]
@manuFL Yes, currently inherited roles work only for selects. We're working on enhancing inherited roles to support mutations and permissions on remote schemas and actions as well. We expect this to be released in the next 2-3 weeks, since this is active work in progress.

Here's a link to the spec we're implementing: https://github.com/hasura/graphql-engine/issues/6991
Please do feel free to add in some notes about your use-case too!
